Source/WTF/wtf/glib/malloc_heap_breakdown/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+set_property(DIRECTORY . PROPERTY FOLDER "malloc_heap_breakdown")
+
+set(MallocHeapBreakdown_SOURCES
+    main.cpp
+)
+
+set_source_files_properties(
+    main.cpp
+    PROPERTIES LANGUAGE CXX
+)
+
+set(MallocHeapBreakdown_PUBLIC_HEADERS
+    malloc.h
+)
+
+set(MallocHeapBreakdown_LIBRARIES
+    ${CMAKE_DL_LIBS}
+)
+
+set(MallocHeapBreakdown_COMPILE_OPTIONS
+    -Wno-cast-align
+    -Wno-missing-field-initializers
+)
+
+set(MallocHeapBreakdown_INTERFACE_LIBRARIES MallocHeapBreakdown)
+set(MallocHeapBreakdown_INTERFACE_INCLUDE_DIRECTORIES ${MallocHeapBreakdown_FRAMEWORK_HEADERS_DIR})
+set(MallocHeapBreakdown_INTERFACE_DEPENDENCIES MallocHeapBreakdown_CopyHeaders)
+
+WEBKIT_FRAMEWORK_DECLARE(MallocHeapBreakdown)
+WEBKIT_INCLUDE_CONFIG_FILES_IF_EXISTS()
+
+WEBKIT_COPY_FILES(MallocHeapBreakdown_CopyHeaders
+    DESTINATION ${MallocHeapBreakdown_FRAMEWORK_HEADERS_DIR}/malloc
+    FILES ${MallocHeapBreakdown_PUBLIC_HEADERS}
+    FLATTENED
+)
```

Comment:
Do you really need forwarding headers just for malloc.h? This seems like overkill.